source/style-guide/terminology/terms-for-global-audience.txt

```diff
@@ -305,6 +301,68 @@ If you create "named" users for extended examples or scenarios, use
 names that represent a variety of ethnic backgrounds, genders, and
 locations.
 
+.. _oppressive-language:
+
+Use Non-Oppressive Language
+---------------------------
+
+Language choice deal in connotation. To ignore that ignores the power
+of speech and writing. Consistent with our goal of empowering everyone
+to use our technology, we must consider our use of terms that have been
+identified as racist, sexist, or otherwise oppressive. Some terms lack
+any ambiguity as to whether or not they are racist.
```

Comment:
This feels a little unnecessarily obfuscated. How about:
```suggestion
Consistent with our goal of empowering everyone
to use our technology, we must consider the 
connotation of the words we use and avoid terms 
that have been identified as racist, sexist, or 
otherwise oppressive. Some terms lack any ambiguity 
as to whether or not they are racist.
```

Reply:
Still a bit wordy. Substituting:
```
Language choice deals in connotation. To ignore that ignores the power of speech and writing. MongoDB wants to empower everyone to use our technology. Keeping with that goal, we must consider our use of terms identified as racist, sexist, or otherwise oppressive. Some terms lack any ambiguity; they're racist. 
```

Reply:
I don't see this change?